lib/Store/CouchDB.pm

```diff
@@ -1,11 +1,13 @@
 package Store::CouchDB;
 
+use strict;
+use warnings;
```

Comment:
`Any::Moose` adds those already. this is unnecessary.

however `Any::Moose` should be replaced with something smaller and faster like `Mouse` because we don't make use of many Moose like features.